### ISSUE TITLE
Empty <h5></h5> elements occurring on the homepage are not focusable.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7176]
### Where To Find Bug
Home page

### Discussion
The screen reader should announce the heading level and its content. There are not currently any of these elements present that a user might wish to focus on. However, in the future there might be, due to the way the <Box> component would handle a `subtitle` prop. Then, the empty heading element would not receive focus, so the screen reader would not announce it. The AT user would not know that it exists on the page.

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)

**Comment by benjiwheeler** [comment]
Paul, I had trouble with these steps to reproduce, because it's not clear to me that there are h5 elements there in the first place. Is there an example of an h5 element where I could see/hear the problem?

**Comment by benjiwheeler** [comment]
We discussed this and agreed that there's no user-facing problem here currently. There are no reproduction steps that would allow a problem to be experienced. But, there could be in the future.
